detection_rules/rule.py

```diff
@@ -759,6 +790,46 @@ def compare_field_versions(min_stack: Version, max_stack: Version) -> bool:
         max_stack = max_stack or current_version
         return Version(min_stack) <= current_version >= Version(max_stack)
 
+    def get_packaged_integrations(self, ast_obj: Union[kql.ast.Expression,
```

Comment:
Ideally, you should iterate the tree as a whole for resiliency. Let's discuss this method in the DED zoom